ui/packages/catalog-ui-search/src/main/webapp/component/visualization/histogram/histogram.view.js

```diff
@@ -309,7 +312,10 @@ module.exports = Marionette.LayoutView.extend({
       )
       this.handleResize()
     } else {
-      this.el.querySelector('.histogram-container').innerHTML = ''
+      const container = this.el.querySelector('.histogram-container')
+      if (container) {
+        container.innerHTML = ''
```

Comment:
What is this line for?

Reply:
Sometimes `this.el.querySelector('.histogram-container')` returns undefined. This is probably due to the `display: none` and `display: block` switching. This is a pre-existing bug. I'm not certain why it sets the innerHTML but it appears to be for the Plotly.js chart.